src/main/java/com/kodcu/provider/MongoToElasticProvider.java

```diff
@@ -56,7 +56,7 @@ public List buildJSONContent(int skip, int limit) {
     private MongoCursor<Document> getCursor(int skip) {
         if (cursor == null && cursorId == 0) {
             Document query = Document.parse(config.getMongo().getQuery());
-            BasicDBObject sort = new BasicDBObject("$natural", -1);
+            BasicDBObject sort = new BasicDBObject("$natural", 1);
```

Comment:
This makes sure that data it is fetched according to the way it exists on the disk. I think mongodb normally uses natural order anyway but couldn't find that comment in the documentation. The actual behavior is undefined so people often decide to specify natural order. Changing this from -1 to 1 didn't seem to change the speed much on our installation, that might be because we use SSDs. I also didn't see much of a change if I removed the line completely.